multicluster-config/workload-values.yaml

```diff
@@ -33,16 +33,27 @@ kube-prometheus-stack:
     enabled: true
   grafana:
     enabled: false
-  prometheus: 
+  prometheus:
     prometheusSpec:
       externalLabels:
         # !IMPORTANT: each cluster must have prometheus external label 'cluster' with unique value.
         # This will be used as cluster discriminator when metrics are aggregated on observer cluster.
         cluster: "workload-cluster"
       replicas: 2
       thanos:
-        objectStorageConfig: 
+        objectStorageConfig:
           name: thanos-objstore-config
           key: objstore.yml
-    thanosService: 
+    thanosService:
       enabled: true
+openshift:
+ # Keep only the service accounts used by workload values
+  monitoringServiceAccounts:
+    - dnation-kubernetes-jsonnet-translator
```

Comment:
Please name it like other serviceaccounts without `dnation`, because now the name is `dnation-monitoring-dnation-kubernetes-jsonnet-translator`